ASIM/lib/functions/ASIM_GetUserType.yaml

```diff
@@ -0,0 +1,46 @@
+Function:
+  Title: An ASIM function sets the UserType for Windows systems based on the username and sid
+  Version: '0.3'
+  LastUpdated: May 1st 2023
+References:
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: KQL Invoke operator
+  Link: https://docs.microsoft.com/azure/data-explorer/kusto/query/invokeoperator
+- Title: Well-Known SID Structures
+  Link: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/81d92bba-d22b-4a8c-908a-554ab29148ab
+- Title: Well-known SIDs
+  Link: https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers#well-known-sids
+Description: |
+  This ASIM tabular function is intended for use in ASIM parsers and sets the UserType in Windows systems based on the username and sid
+FunctionName: ASIM_GetUserType
+EquivalentBuiltInFunction: _ASIM_GetUserType
+FunctionParams:
+  - Name: username
+    Type: string
+  - Name: sid
+    Type: string
+
+FunctionQuery: |
+  let ASIM_GetWindowsUserType = (username:string, sid:string) { 
```

Comment:
Change the name here and in the calling line at the end as well.

Reply:
Done.